docker_explorer/storage.py

```diff
@@ -202,10 +186,13 @@ def MakeMountCommands(self, container_object, mount_dir):
       lower_dir = self._BuildLowerLayers(lower_fd.read().strip())
     upper_dir = os.path.join(mount_id_path, self.UPPERDIR_NAME)
 
-    cmd = [
+    commands = [[
         '/bin/mount', '-t', 'overlay', 'overlay', '-o',
-        'ro,lowerdir={0:s}:{1:s}'.format(upper_dir, lower_dir), mount_dir]
-    return [cmd]
+        'ro,lowerdir={0:s}:{1:s}'.format(upper_dir, lower_dir), mount_dir]]
+
+    # Adding the commands to mount any extra declared Volumes and Mounts
+    commands.extend(self._MakeVolumeMountCommands(container_object, mount_dir))
+    return commands
```

Comment:
same as the one before, the return type in the doc-string needs to be corrected:
I believe the return type in this method is list of lists not list of strings, please look into the doc-string of this method and change as needed.

Reply:
done